pyproject.toml

```diff
@@ -1,2 +1,63 @@
+# https://packaging.python.org/en/latest/tutorials/packaging-projects/
+# https://hatch.pypa.io/latest/config/build/
+
+[project]
+name = "tutor-notes"
+description = "A Tutor plugin for student notes"
+authors = [
+  { name = "Edly" },
+  { email = "hello@edly.io"},
+]
+maintainers = [
+  { name = "eduNEXT" },
```

Comment:
Shouldn't this have your name here @jfavellar90 instead of eduNEXT?

Reply:
Fixed